src/load.ts

```diff
@@ -1,23 +1,9 @@
-import type FpJs from '@fingerprintjs/fingerprintjs-pro'
-import type { LoadOptions, GetResult } from '@fingerprintjs/fingerprintjs-pro'
+import FingerprintJS from '@fingerprintjs/fingerprintjs-pro'
```

Comment:
Consider using `* as FingerprintJS` instead of `FingerprintJS` to let Rollup exclude the unused parts of the JS agent code from the distributive bundles.